content/terraform-docs-common/docs/cloud-docs/integrations/service-now/service-catalog-terraform/developer-reference.mdx

```diff
@@ -13,16 +13,16 @@ using the information found in this document.
 ServiceNow has the concept of a Variable Set which is a collection of ServiceNow
 Variables that can be referenced in a Flow from a Service Catalog item. The
 Terraform Integration codebase can create [Terraform Variables and Terraform
-Environment Variables](/terraform/cloud-docs/variables) via the API using the
+Environment Variables](/terraform/cloud-docs/workspaces/variables) via the API using the
 `tf_variable.createVariablesFromSet()` function.
 
 This function looks for variables following these conventions:
 
 | ServiceNow Variable Name         | HCP Terraform Variable                                   |
 | -------------------------------- | ---------------------------------------------------------- |
-| `tf_var_VARIABLE_NAME`           | Terraform Variable: `VARIABLE_NAME`                        |
+| `tf_var_hcl_VARIABLE_NAME`           | Terraform Variable: `VARIABLE_NAME`                        |
```

Comment:
Are these variable names changing? Is this a breaking change if someone is on a previous version? Or does ServiceNow automatically keep this integration up to date?

Reply:
No, not breaking changes - it just wasnt documented